docker/0.20.0/base/Dockerfile.cpu

```diff
@@ -1,32 +1,25 @@
 FROM ubuntu:16.04
 
-ARG py_version
-
-# Validate that arguments are specified
-RUN test $py_version || exit 1
-
 # Install python and other scikit-learn runtime dependencies
 # Dependency list from http://scikit-learn.org/stable/developers/advanced_installation.html#installing-build-dependencies
 RUN apt-get update && \
-    apt-get -y install build-essential libatlas-dev git wget curl nginx jq && \
-    if [ $py_version -eq 2 ]; \
-       then apt-get -y install python-dev python-setuptools \
-                     python-numpy python-scipy libatlas3-base; \
-       else apt-get -y install python3-dev python3-setuptools \
-                     python3-numpy python3-scipy libatlas3-base; fi
+    apt-get -y install build-essential libatlas-dev git wget curl nginx jq libatlas3-base
+
+RUN curl -LO http://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh && \
+    bash Miniconda3-latest-Linux-x86_64.sh -bfp /miniconda3 && \
+    rm Miniconda3-latest-Linux-x86_64.sh
+
+ENV PATH=/miniconda3/bin:${PATH}
 
-# Install pip
-RUN cd /tmp && \
-     curl -O https://bootstrap.pypa.io/get-pip.py && \
-     if [ $py_version -eq 2 ]; \
-        then python2 get-pip.py; \
-        else python3 get-pip.py; fi && \
-     rm get-pip.py
+RUN conda update -y conda && \
+    conda install protobuf tbb libiconv && \
```

Comment:
You don't need this line. `mlio-py` installs them as its dependencies.

Reply:
done